src/run/port.rs

```diff
@@ -65,6 +68,14 @@ bi_enum! {
     /// storing the targets of the wires connected to the two auxiliary ports of
     /// this node.
     Op = 4,
+    /// An `F32` port represents the principal port of an 32-bit floating
+    /// point node.
+    ///
+    /// The top 32 bits of the port are the value of this node and is
```

Comment:
It's not the top 32 bits, but rather the low 32 bits of the top 60 bits, right? (not that that's a particularly clear way to describe it...)